resume.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title> Rossy </title>
+        	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap" rel="stylesheet">
+    		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      		<style>
+                  body {
+    padding-left: 10%;
+    padding-right: 10%;
+    margin-bottom: 10%;
+    background-color: #c3fbf4;
+}
+.Sticky {
+    width: 20vw;
+    height: 30vh;
+    position: fixed;
+    top: 0px;
+    z-index: 0;
+
+}
+.Main {
+    margin-top: 45vh;
+    background-color: #FF6978;
+    width: 80vw;
+    height: relative;
```

Comment:
`relative` is not a valid value for the property `height`.
```suggestion
```
Dev tools in Firefox will show this to you:
<img width="369" alt="Screen Shot 2020-10-04 at 3 11 29 PM" src="https://user-images.githubusercontent.com/2653980/95024905-f48f0d00-0653-11eb-9bab-bd5fdcff2fe9.png">